taxonomy/models.py

```diff
@@ -458,6 +460,7 @@ class Job(TimeStampedModel):
             'The title of job.'
         )
     )
+    description = models.TextField(default='', help_text='AI generated job description.')
```

Comment:
shouldn't this field have a max character length? Unsure how much long texts can be indexed into Algolia.

Reply:
This will be small text. Around 50 words